package.json

```
@@ -22,7 +22,8 @@
   "exports": {
     ".": {
       "import": "./lib/index.mjs",
-      "require": "./lib/index.js"
+      "require": "./lib/index.js",
+      "types": "./types/index.d.ts"
     },
     "./_non-semver-use-at-your-own-risk_/*": "./lib/*",
```

this needs to be updated too

```suggestion
    "./_non-semver-use-at-your-own-risk_/*": {
      "default": "./lib/*",
      "types": "./types/*.d.ts"
    },
```

---

updated